### ISSUE TITLE
Question about Released Checkpoint for User Prompts

### ISSUE THREAD

**Issue by SouLeo** [collinzrj/output2prompt#5]
Hello!

Thank you for your work and excellent repo. I am using this work heavily in a study I am performing, but I am having difficulties with the `chat_prompt_stealing_good` checkpoint downloadable here: 

`wget "https://zenodo.org/records/12759549/files/prompt2output_inverters.zip?download=1" -O prompt2output_inverters.zip`

Specifically, when I use this model on inversions on the published training set: `train/chat_instruction2m` from here `wget "https://zenodo.org/records/12759549/files/prompt2output_datasets.zip?download=1" -O prompt2output_datasets.zip` I get some inversions that feel semantically similar, but are not exact. I'm wondering if that is expected on the _training_ dataset.

From your ACL paper, I see this:

> We ... train the models for 3 epochs using the Adam optimizer at a constant learning rate of 2e-4. 

But I guess I'm uncertain what the means in terms of performance on the training dataset (F1/BLEU).

Anyway, here are some example predictions using that T5 sparse checkpoint on the training dataset:

```json
{
  "original_prompt": "Which is an appropriate title for this article? Global warming is threatening travel destinations worldwide. What's more, travelers themselves are contributing to it. Find out what you can do about it. OPTIONS: - World - Sports - Business - Science/Tech",
  "inv_prompt": "What is an appropriate title for this article? global warming is affecting travel destinations worldwide and how travelers themselves are contributing to it."
}
```

```json
{
  "original_prompt": "Given the fact that: The heavier a weight you lift, the stronger your muscles will become. Instruction: If you want to gain muscle strength, should you lift heavy or light weights?",
  "inv_prompt": "What is the principle of \"to gain muscle strength\"?"
}
```

```json
{
  "original_prompt": "Use evidence from the fact that erosion of soil has a negative impact on the environment to answer this question: \"The running and moving liquid fluid during a storm causes what\" OPTIONS: - insects - weather damage - abrasion - lava",
  "inv_prompt": "Use evidence from the fact that erosion of soil has a negative impact on the environment to answer the question: \"The running and moving liquid fluid during a storm causes what\""
}
```

Thank you for your time!
